CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Changes History
 
+4.0.3
```

Comment:
This is new functionality, not bugfix, so version is better to be `4.1.0` according to https://semver.org/ rules

Reply:
done